Pages/LoginPage.cs

```diff
@@ -0,0 +1,35 @@
+﻿using OpenQA.Selenium;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace OnboardProjectMars.Pages
+{
+    public class LoginPage
+    {
+        //Function to login to the Mars Portal
```

Comment:
5/
Please implement the POM Pattern, please see the below  for example how it looks you can see the pages how they are set 
https://docs.specflow.org/projects/specflow/en/latest/ui-automation/Selenium-with-Page-Object-Pattern.html


this applies to all pages